.github/workflows/slither.yml

```
@@ -0,0 +1,73 @@
+name: Static-Analysis
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - 'contracts/**'
+      - '.github/workflows/slither.yml'
+  pull_request:
+    branches: [ main ]
+    paths:
+      - 'contracts/**'
+      - '.github/workflows/slither.yml'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6]
+        node-version: [16.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Branch name
+        run: echo running on branch ${GITHUB_REF##*/}
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Create env file
+        run: |
+          cat << EOF > .env
+          MUMBAI_API_URL=""
+          POLYGON_API_URL=""
+          PRIVATE_KEY="0000000000000000000000000000000000000000000000000000000000000000"
+          POLYSCAN_KEY=""
+          EOF
+      - run: npm install
+      - run: npm run build
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install Slither and dependencies
+        run: |
+          sudo snap install solc
+          python -m pip install --upgrade pip
+          pip install slither-analyzer solc-select
+          solc-select install 0.8.2
+          solc-select use 0.8.2
+      - name: Prepare output
+        run: |
+          mkdir -p .github/slither-output
+      - name: Summary of static ananlysis
+        shell: bash
+        run: |
+          echo "Running slither"
+          #slither . --ignore-compile --filter-paths "node_modules" --disable-color --print human-summary >>.github/slither-output/slither_report.txt 2>&1 || true
+          slither . --json-types compilations,console,detectors,printers,list-detectors,list-printers --exclude-dependencies --exclude-informational --compile-force-framework hardhat >>.github/slither-output/slither_report.txt 2>&1 || true
```

```suggestion
          slither . --json-types compilations,console,detectors,printers,list-detectors,list-printers --exclude-dependencies --exclude-informational --compile-force-framework hardhat 2>&1 | tee -a .github/slither-output/slither_report.txt
```

should do the trick and also print it out to stdout at the same time.... (tee always succeeds)

---

But then again.... Do we need this report as a downloadable file? Not that it has become so short.... just printing it out should be enough - and if the issue with the warning count mentioned in my comment above is dealt with.. then the suppression of the error status code will also become unnecessary